Framework/Algorithms/inc/MantidAlgorithms/ReflectometrySumInQ.h

```diff
@@ -43,12 +43,20 @@ namespace Algorithms {
 */
 class MANTID_ALGORITHMS_DLL ReflectometrySumInQ : public API::Algorithm {
 public:
+  struct Angles {
+    double horizon{std::nan("")};
```

Comment:
Please change this to [`std::numeric_limits<double>::quiet_NaN()`](https://en.cppreference.com/w/cpp/types/numeric_limits/quiet_NaN).